test/extensions/filters/http/jwt_authn/filter_integration_test.cc

```diff
@@ -1,10 +1,16 @@
 #include "envoy/config/filter/http/jwt_authn/v2alpha/config.pb.h"
 
+#include "common/router/string_accessor_impl.h"
+
+#include "extensions/filters/http/common/empty_http_filter_config.h"
+#include "extensions/filters/http/common/pass_through_filter.h"
 #include "extensions/filters/http/well_known_names.h"
 
 #include "test/extensions/filters/http/jwt_authn/test_common.h"
 #include "test/integration/http_protocol_integration.h"
 
+#include "fmt/printf.h"
```

Comment:
this should be included via common/common/fmt.h?

Reply:
Not needed if use absl::StrCat

Reply:
That SG too.